src/sugar3/graphics/style.py

```diff
@@ -46,39 +52,75 @@ def _compute_zoom_factor():
 
 
 class Font(object):
-
+	'''
+	A font defines the style of how the text should be rendered. 
+	
+	Args:
+		desc (str): a description of the Font object
+	'''
     def __init__(self, desc):
         self._desc = desc
 
     def __str__(self):
+		'''
+		Returns description of font
+		'''
         return self._desc
 
     def get_pango_desc(self):
+		'''
+		Returns Pango description of font
+		'''
         return Pango.FontDescription(self._desc)
 
 
 class Color(object):
-
+	'''
+	A Color object defines a specific color. 
+
+	Args:
+		color (str): String in the form #FFFFFF representing the color
+		
+		alpha (double):  transparency of color
+	'''
     def __init__(self, color, alpha=1.0):
         self._r, self._g, self._b = self._html_to_rgb(color)
         self._a = alpha
 
     def get_rgba(self):
+		'''
+		Returns 4-tuple of red, green, blue, and alpha levels
+		'''
         return (self._r, self._g, self._b, self._a)
 
     def get_int(self):
+		'''
+		Returns color encoded as an int, in the form rgba
+		'''
         return int(self._a * 255) + (int(self._b * 255) << 8) + \
             (int(self._g * 255) << 16) + (int(self._r * 255) << 24)
 
     def get_gdk_color(self):
+		'''
+		Returns GDK standard color
```

Comment:
How ould I do that with sphinx? Would it just be :class:Gdk.Color on a new line?